### ISSUE TITLE
Feature (RW-Valet Annual): Update folder status with renewal (Ready for UAT on PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#9969]
Acceptance Criteria (Permit Renewal)

The following steps were performed in RW-Valet Parking Annual Folder for Renewal
-- A bill was created for fees to be paid
-- The fees in the folder were paid (through portal or Back Office)
-- In the Annual Valet Permitting Process, Process info fields were updated with new Issue date and new Expiry Date
-- Attempt 'Update issue and expiry dates' was made in the annual valet permitting process, along with attempt comments.

Expected result: Folder should update folder status from 'Expired' to 'Active', and also update issue and expiry dates based on process info.

Actual result: Folder updated issue and expiry dates, but did not update folder status to 'Active'

**Comment by SubhaShroff** [comment]
@TracyLinder @Nadin-Nader  This is deployed in SB and tested. Please let me know if it can be deployed in DEV.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  DEV deployment request is submitted - SCTASK0243501 (70c49b7dc2f1)

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  DEV deployment is done. Tested it on DEV BO and it is working as expected.
RSN: 12516761

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I have submitted SCTASK0244167 (70c49b7dc2f1) for TEST deployment.

**Comment by SubhaShroff** [comment]
@TracyLinder  @Nadin-Nader  TEST Deployment is done and tested it for RSN: 12870111. 
After adding, attempt 'Update issue and expiry dates' was made in the annual valet permitting process, along with issue date, expiry date; the folder status also changed to ACTIVE.
This is ready for UAT.

**Comment by SubhaShroff** [comment]
### Status:
Waiting for the UAT.

**Comment by Nadin-Nader** [comment]
Approved for PROD

**Comment by SubhaShroff** [comment]
@TracyLinder  @Nadin-Nader  This ticket is deployed in PROD.

**Comment by Nadin-Nader** [comment]
Sent a message to Paloma about UAT for this. Waiting on Response

**Comment by TracyLinder** [comment]
We tested in PROD, works great.

**Comment by Nadin-Nader** [comment]
This has been fixed
